com/AlterETM.py

```diff
@@ -222,8 +210,7 @@ def insert_modify_param(parser, cnn, stnlist, args):
 
                     if sdate[0] <= jump.date <= sdate[1]:
                         # this jump should be removed, fill in the data
-                        tpar.Year = jump.date.year
```

Comment:
@demiangomez flake8 is complaining the `jump` variable isn't defined, so this might be a bug:

```
AlterETM.py:229:47: F821 undefined name 'jump'
AlterETM.py:229:63: F821 undefined name 'jump'
```

Reply:
I use flake8 version 7.1.1 (newest) and don´t show that line.

Reply:
great, thanks!